packages/contentstack-migration/examples/05-Update-reference-entry-from-mapper.js

```diff
@@ -0,0 +1,138 @@
+const fs = require('fs');
+const chalk = require('chalk');
+module.exports = async ({ migration, stackSDKInstance, managementAPIClient, config }) => {
+  const modules = ['entries', 'assets', 'extensions'];
+
+  const readAllModulesUids = (filePath) => {
+    let uidMapping = [];
+    modules.forEach((module) => {
+      if (fs.existsSync(`${filePath}/mapper/${module}/uid-mapping.json`)) {
+        let mappedIds = fs.readFileSync(`${filePath}/mapper/${module}/uid-mapping.json`, 'utf-8');
+        mappedIds = JSON.parse(mappedIds);
+        uidMapping = { ...uidMapping, ...mappedIds };
+      } else {
+        console.log(chalk.red(`Mapper Does not exist for module '${module}'`));
+      }
+    });
+
+    return uidMapping;
+  };
+
+  const getEntries = async (ct) => {
+    try {
+      let entries = [];
+      let skip = 0;
+      let limit = 1;
```

Comment:
@cs-raj , You can choose a maximum limit of 100 as an alternative to 1. This would prevent pointless API calls. For example, an API call would be made 60 times if a single CT had 60 entries. You can also provide the limit and skip value through config if required.

Reply:
Thanks @aman19K For pointing this out I actually missed to change it to original values